frontend/src/components/VModal/VModalContent.vue

```diff
@@ -122,7 +122,9 @@ defineExpose({
         v-bind="$attrs"
         class="flex flex-col"
         :class="[
-          mode === 'dark' ? 'bg-black text-default' : 'bg-overlay text-default',
+          mode === 'dark'
+            ? 'dark-mode bg-black text-default'
```

Comment:
The `dark` mode `VModalContent` is manually set to `dark-mode` so that the focus rings are yellow and don't change with the sites main color mode.